### ISSUE TITLE
release time

### ISSUE THREAD

**Issue by aspire5515** [Sheights/StarboundSimpleVoreMod#187]
is there a set date, or even an estimation, about when the next update will come out? it has been over a year since the last one and I am starting to think this mod got dropped...

**Comment by florancetheavian** [comment]
I don't believe it was dropped. from what I heard there is not a very big team working on the mod itself, therefore it takes time to update it. I personally have no problems with the amount of time it takes to update, I just enjoy the mod. does any of this help?
